phoenix-ios/phoenix-ios/views/onboarding/InitializationView.swift

```diff
@@ -142,8 +142,10 @@ struct InitializationView: MVIView {
 		let swiftEntropy = AppSecurity.shared.generateEntropy()
 		let kotlinEntropy = swiftEntropy.toKotlinByteArray()
 		
-		let intent = Initialization.IntentGenerateWallet(entropy: kotlinEntropy)
-		mvi.intent(intent)
+		mvi.intent(Initialization.IntentGenerateWallet(
+			entropy: kotlinEntropy,
+			language: MnemonicLanguage.defaultCase
```

Comment:
This `defaultCase` depends on the user's preferences within the OS, and so it maybe end up being Czech, English, French, or Spanish.

However it seems to me that users nowadays would expect having an English seed by default. I've never had any localized seed for example, but I may not be a good example. What I mean is that some people may be confused if they don't have the regular English words in their seed. 

Also, I don't know if other wallets support non-English seeds? For example, on my current version of Electrum using French as the GUI language, I cannot create a French seed. This would be an issue if people want to recover on-chain funds after a force-close (as mentioned in https://phoenix.acinq.co/faq#my-channels-got-force-closed-how-do-i-recover-my-money).

What do you think about this?

In any case I think it would be a good idea to let people choose what language they use before creating the seed.  The seed initialization screen would need an additional step where people pick a language.

Reply:
Mitigated with https://github.com/ACINQ/phoenix/pull/233#issuecomment-1010167040